lms/templates/js/discussion/thread-list-item.underscore

```diff
@@ -0,0 +1,97 @@
+<li data-id="<%- id %>" class="forum-nav-thread<% if (typeof(read) != "undefined" && !read) { %> is-unread<% } %>">
+  <a href="#" class="forum-nav-thread-link">
+    <div class="forum-nav-thread-wrapper-0">
+      <%
+      var icon_class, sr_text;
+      if (thread_type === "discussion") {
+          icon_class = "fa-comments";
+          // Translators: This is a label for a Discussion forum thread
+          sr_text = gettext("discussion");
+      } else if (endorsed) {
+          icon_class = "fa-check-square-o";
+          // Translators: This is a label for a Question forum thread with a marked answer
+          sr_text = gettext("answered question");
+      } else {
+          icon_class = "fa-question";
+          // Translators: This is a label for a Question forum thread without a marked answer
+          sr_text = gettext("unanswered question");
+      }
+      %>
+      <span class="sr"><%= sr_text %></span>
```

Comment:
This span seems to be missing in the resulting DOM. It also appears to be a problem on prod, so it's not a regression, but I thought it was worth noting.